vault/dendron.tutorial.conclusion.md

```diff
@@ -11,23 +11,62 @@ nav_order: 4
+    - Learn how to navigate quickly to related notes using [[Go to Commands|dendron://dendron.dendron-site/dendron.topic.navigation#go-to]].
+- Organizing
+    - Learn to expand your organization beyond hierarchies by [[using tags|dendron://dendron.dendron-site/dendron.topic.tags]].
+    - Learn how to [[tag other Users|dendron://dendron.dendron-site/dendron.topic.users]] in your notes.
+    - Learn how to [[manage and track tasks with special Task Notes|dendron://dendron.dendron-site/dendron.topic.tasks]].
+    - Learn how to quickly take notes that may note have a home quite yet, like a scratchpad, using [[Scratch Notes|dendron://dendron.dendron-site/dendron.topic.special-notes#scratch-note]].
+    - Learn how to log your activities with [[Daily Journal Notes|dendron://dendron.dendron-site/dendron.topic.special-notes#daily-journal]] and other generic [[Journal Notes|dendron://dendron.dendron-site/dendron.topic.special-notes#journal-note]].
+    - Learn the _how_ and the _why_ behind [[Hierarchies in Dendron|dendron://dendron.dendron-site/dendron.topic.hierarchies]].
+    - Learn how to automatically make use of [[Templates|dendron.topic.templates]] in new notes, in specific hierarchies, with [[Schemas|dendron://dendron.dendron-site/dendron.topic.schema]].
+- Sharing
+    - Learn how to backup your vaults online, privately or accessible to the public, with [[git and GitHub|dendron://dendron.dendron-site/dendron.topic.git]].
+
+### Forest
+
+- Retrieval
+    - Learn how to customize the power and functionality of your lookups with [[Lookup Modifiers|dendron://dendron.dendron-site/dendron.topic.lookup.modifiers]].
+- Editing
+    - Learn how to understand and extend the metadata of your notes with [[Frontmatter Attributes|dendron://dendron.dendron-site/dendron.topic.frontmatter]].
+- Organizing
+    - Learn how to [[manage and understand different kinds of Vaults|dendron://dendron.dendron-site/dendron.topic.vaults]].
+    - Learn how to [[manage and understand different kinds of Workspaces|dendron://dendron.dendron-site/dendron.topic.workspace]].
+    - Learn how to work with [[Multiple Vaults in the same workspace|dendron://dendron.dendron-site/dendron.topic.multi-vault]]. This allows you to access private personal notes from a vault, while working with notes from other vaults (like team notes, or software documentation in a project repository).
+- Sharing
+    - Learn how to [[publish your notes online|dendron://dendron.dendron-site/dendron.topic.publish.quickstart]], and follow guides for hosting with [[Netlify|dendron.topic.publish.cook.netlify]] or [[GitHub Pages|dendron.topic.publish.cook.github]].
+    - Learn how to [[access your Dendron notes on mobile devices with third-party software|dendron.guides.cook#mobile-support]].
+- Transfer
+    - Learn how to use Dendron interoperably with other formats, by [[exporting/importing your notes with Pods|dendron://dendron.dendron-site/dendron.topic.pod]].
+
+### Wilderness
```

Comment:
This is for people that want to explore the cutting edge of Dendron. Explore here to master advanced functionality in Dendron